app/assets/javascripts/miq_flash.js

```diff
@@ -46,12 +61,27 @@ function add_flash(msg, level, options) {
   }
 
   $('#flash_msg_div').append(textDiv).show();
+
+  // Check if text is *not* overflowing and options.long_alert is true thus disabling the button.
+  if ((alertDiv[0].scrollWidth <= alertDiv.innerWidth()) && options.long_alert) {
```

Comment:
Is this bit strictly necessary?

Because, if we do that, we need to watch for `resize` events and recompute...

Reply:
@himdel we can remove this altogether if we don't mind having a dangling button...  There is a case when we pass `long_alert => true` but the actual text is not large than we would have a button that does nothing...
If we wish to solve this case - I can try and find a different solution if you're not comfortable with this one.

Reply:
I'm just worried that if we do it, we'll get BZs that it won't work when the window gets resized :).

But you're right that if we just let the button active, we may equally well get BZs that say the button does nothing :).

Unless somebody has a strong opinion here, maybe just leave it as is and we'll see what happens.